core/src/tvu.rs

```diff
@@ -451,7 +454,7 @@ pub mod tests {
             None,
             None,
             AbsRequestSender::default(),
-            false, // use_quic
+            &Arc::new(ConnectionCache::new(DEFAULT_TPU_USE_QUIC)),
```

Comment:
Since `DEFAULT_TPU_USE_QUIC` is defined in `solana-send-transaction-service`, and not in `solana-client`, it made sense to use the default from there.  Is the idea to eventually change `ConnectionCache::default()` to use quic rather than change `DEFAULT_TPU_USE_QUIC` to true?  If so, then I'll change all of these to `ConnectionCache::default()`

Reply:
Yes, with the default(), we should and could minimize usage of DEFAULT_TPU_USE_QUIC externally.

Reply:
It looks much cleaner, thanks for the suggestion!